components/movementsensor/gpsrtk/gpsrtkpmtk_test.go

```diff
@@ -0,0 +1,55 @@
+//go:build linux
```

Comment:
might want to rename this file `pmtk_test` and the `gpsrtkserial_test` -> `serial_test`